geocoder/ip-addresses/test/data/test.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/sh
+
+# 
+# SQL_API_ROUTE and API_KEY need to be
+# conveniently filled for the requests
+# to work.
+#
+
+
+SQL_API_ROUTE="****"
+API_KEY="*****"
```

Comment:
comment for the future: these parameters can be fed from command line options, env variables or a config file instead of requiring the source file to be modified.